sdk/storage/test/blobs/performance_benchmark.cpp

```diff
@@ -36,4 +43,69 @@ namespace Azure { namespace Storage { namespace Test {
     }
   }
 
+  TEST_F(BlobContainerClientTest, DISABLED_MultiThreadPerf)
+  {
+    constexpr int concurrency = 64;
+    std::vector<Blobs::BlockBlobClient> blockBlobClients;
+    for (int i = 0; i < concurrency; ++i)
+    {
+      blockBlobClients.emplace_back(
+          Azure::Storage::Blobs::BlockBlobClient::CreateFromConnectionString(
+              StandardStorageConnectionString(),
+              m_containerName,
+              "MultiThreadPerf" + RandomString()));
+    }
+
+    constexpr std::size_t bufferSize = static_cast<std::size_t>(1_GB);
+    std::vector<uint8_t> buffer = RandomBuffer(bufferSize);
+
+    {
+      std::vector<std::future<void>> futures;
+      Blobs::UploadBlobOptions options;
+      options.ChunkSize = 8_MB;
```

Comment:
This number should be tuned in the future. 8MB is fine for this release.

Reply:
We can never find a number suitable for all kinds of hardware configuration. So you tune it for a specific machine, it still won't work for other machines.